lib/page/set-multipart-form/set-multipart-form.unit.spec.js

```diff
@@ -27,7 +28,7 @@ test('When setMultipartForm is passed a page containing upload controls', async
   })
 
   t.equal(pageInstance.encType, true, 'it should set page’s encType to true')
-  t.equal(pageInstance.components[0].maxSize, 10 * 1024 * 1024, 'it should set component’s maxSize')
+  t.equal(pageInstance.components[0].maxSize, setUploadControlsMaxSize.defaultMaxSize, 'it should set component’s maxSize')
```

Comment:
👍 
`10 * 1024 * 1024` just looked super weird to me

Reply:
NB. this change isn't about it looking super weird, but decoupling it from whatever the default maxsize happens to be